bindings/nodejs/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "nodejs"
+version = "0.2.0"
+description = "Node.js binding to the wallet library"
+authors = ["Antoni Dikov <antoni.dikov@iota.org>"]
+license = "MIT"
+edition = "2018"
+exclude = ["index.node"]
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies.neon]
+version = "0.9"
+default-features = false
+features = ["napi-6", "event-queue-api"]
+
+[dependencies]
+iota-wallet = { path = "../../", version = "0.2.0", features = ["stronghold"] }
+iota-client = { git = "https://github.com/iotaledger/iota.rs", rev = "57c5a33d83be4b8286a54156e9f0d2ac4345e9b9", features = ["mqtt"] }
```

Comment:
That's only used for the logger I think, would be better to only import bee-common then